.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,170 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - dev
+      - rb/parallelize-ci-workflow
```

Comment:
Reminder to remove before merging.

Reply:
Do you need to remove this?

Reply:
@jharenza yep, I'll do that before merge as long as there are no other changes you request in your review.